.github/CHANGELOG.md

```diff
@@ -98,6 +98,10 @@
   required to fully support end-to-end differentiable Mottonen and Amplitude embedding.
   [(#922)](https://github.com/PennyLaneAI/pennylane/pull/922)
 
+* Adds the following generators to the `qml.operation` module: `MultiRZ`, `PauliRot`. These 
+  generators are required to construct the metric tensor for some optimizer. 
+  [(#895)](https://github.com/PennyLaneAI/pennylane/pull/895)
```

Comment:
```suggestion
* Adds the `PauliRot` generator to the `qml.operation` module. This 
  generator is required to construct the metric tensor. 
  [(#963)](https://github.com/PennyLaneAI/pennylane/pull/963)
```
Thanks! Removing the mention of `MultiRZ` since we added a line for that in the previous release.

Reply:
no problem